server/index.js

```diff
@@ -37,7 +38,8 @@ server.connection({
 
 server.register([
   HapiAuthCookie,
-  Bell
+  Bell,
+  Lout
```

Comment:
Not sure about this bit right here.
I don't know if we should just blindly include Lout always, or if we want to wrap this in a `config.get('env')` and only include the /docs route if we're not in "production" environment.